entries/rgb-grafx-demo/game.json

```diff
@@ -0,0 +1,17 @@
+{
+    "date": "2000-01-17",
+    "files": [
+        {
+            "default": true,
+            "filename": "RGB Grafx Demo (PD) [C].gbc",
+            "playable": true
+        }
+    ],
+    "platform": "GBC",
+    "screenshots": [
+        "RGB Grafx Demo (PD) [C].png"
```

Comment:
I think this one should be retaken manually

Reply:
Last I checked, that was literally all the demo was